web/source/kmwembedded.ts

```diff
@@ -525,7 +525,7 @@ namespace com.keyman.text {
     try {
       // Now that we've manually constructed a proper keystroke-sourced KeyEvent, pass control
       // off to the processor for its actual execution.
-      return keyman.core.processKeyEvent(Lkc) != null;
+      return !keyman.core.processKeyEvent(Lkc);
```

Comment:
After a more thorough investigation, LGTM now.  That said... please add a bit of documentation to make this clearer for the future:

```suggestion
      // Should return `false` when Keyman _has_ fully handled the event and `true` when not.
      return !keyman.core.processKeyEvent(Lkc);
```

Reply:
I've added comments about the expected return values.

But still might have some loose ends:
1. While retesting, I noticed K_SPACE also has no effect in the Keyman Text area.
2. After KMW passes K_TAB / K_ENTER back to KMEA to dispatch the key, should I also call KMManager.resetContext() afterwards?

This still wouldn't handle the scenario in #3662 where "Clear Text" updates the KMW text to `""`

Reply:
> 2\. After KMW passes K_TAB / K_ENTER back to KMEA to dispatch the key, should I also call KMManager.resetContext() afterwards?

Ideally, that should be resolved by tracking context changes made by the app or other systems.

Reply:
> While retesting, I noticed K_SPACE also has no effect in the Keyman Text area.

After I did my deeper dive into that part of KMW, I kind of expected this.  It's treated very similarly to K_TAB and K_ENTER internally, as many keyboards rely on default behavior here.

Reply:
Just did up https://github.com/keymanapp/keyman/pull/3707 as a patch to address this issue.